test-app-async/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagenameasync/ReactiveApp.kt

```diff
@@ -6,8 +6,8 @@ import org.springframework.boot.runApplication
 
 @SpringBootApplication
 @ConfigurationPropertiesScan
-class HmppsTemplateKotlin
+class ReactiveApp
```

Comment:
Changed the app name in the reactive / non-reactive test applications as it was causing confusion when running tests manually.